src/containers/home/home.component.jsx

```diff
@@ -8,19 +8,46 @@ import Navbar from "containers/navbar";
 
 const Home = (props) => {
   const [posts, setPosts] = useState([]);
-
+  const [displayPosts, setDisplayPosts] = useState([]);
+  const [selectDisplayPostOption, setSelectDisplayPostOption] = useState(
+    "Newest"
+  );
+  const [viewByTimeframeTime, setViewByTimeframeTime] = useState(Date.now());
+  const [viewByTimeframe, setViewByTimeframe] = useState("All-time");
   const userFromState = useSelector((state) => state.users.user);
+  const userVoteHistory = userFromState.vote_history;
   const doesUserHaveUsername = !!userFromState.username;
+  const postsFromState = useSelector((state) => state.posts);
+
   const signOut = () => {
     auth.signOut();
     props.logoutThunk();
   };
-  const postsFromState = useSelector((state) => state.posts);
+
+  const handleDisplayPostChange = () => {
+    const viewOption = document.getElementById("select-view-option").value;
```

Comment:
Lol remember how we never should do this?

Reply:
I just dont know how to get that current value